tests/integration/nmstatectl_show_test.py

```diff
@@ -27,7 +27,7 @@
 RC_SUCCESS = 0
 RC_FAIL2 = 2
 
-LOOPBACK_JSON_CONFIG = b"""        {
+LOOPBACK_JSON_CONFIG = """        {
```

Comment:
How come `u` is not needed for py2? The `decode` returns unicode.

Reply:
No idea who python2 handing it with out failure. But I will convert the return value to string by using `unicode.encode('utf-8')`.

Reply:
I think you need `decode`, that should fit both py2 and py3: Converting binary/bytes to text.